acceptance-tests/features/add_title_number_to_a_case.feature

```diff
@@ -0,0 +1,32 @@
+@add_title_number_to_a_case
```

Comment:
Similar to comment about the same thing [here](https://github.com/LandRegistry/charges-case-api/pull/26)